src/MergeContext.js

```diff
@@ -269,6 +269,8 @@ class Label
 
     needsAdditionToGitHub() { return !this._presentOnGitHub && this._presentHere; }
 
+    onAddedToGitHub() { this._presentOnGitHub = true; }
```

Comment:
For consistency with other similar methods:

```suggestion
    markAsAdded() { this._presentOnGitHub = true; }
```

If it is not difficult, let's also call this new method after pushing any label, probably inside _addToGitHub(). This will require passing the label to that method (as opposed to just the label name), but that is OK.

Reply:
Done.